### ISSUE TITLE
{{ hcaptcha }} Cannot be inserted correctly

### ISSUE THREAD

**Issue by YinBuLiao** [KnugiHK/flask-hcaptcha#8]
![image](https://user-images.githubusercontent.com/35324248/222771719-80f7b95e-ec52-4637-863d-00092f4761d4.png)

**Comment by YinBuLiao** [comment]
After manual insertion, the backend cannot verify if the CAPTCHA challenge is completed

**Comment by KnugiHK** [comment]
Any more details for the reproduction of your situation?

**Comment by Serphentas** [comment]
You ~should~ may want to use `{{ hCaptcha | safe }}` otherwise the script code will be escaped. Also, you (normally) trust whatever code is given to you by this hCaptcha plugin.

**Comment by KnugiHK** [comment]
To clarify. Code from `{{ hCaptcha }}` is provided by flask-hcaptcha (this addon), which is not affiliated with nor endorsed by hCaptcha.

**Comment by Serphentas** [comment]
@KnugiHK that is correct, I changed my comment to reflect this matter more accurately.
